### ISSUE TITLE
Link not found exception is thrown

### ISSUE THREAD

**Issue by Saboteur777** [putyourlightson/craft-campaign#180]
### Describe the bug
Visiting the links from a sent email (e.g.: `actions/campaign/t/click?cid=CID&sid=SID`) ends up in a 404. 

### To reproduce
Steps to reproduce the behaviour:
1. Send a sendout.
2. Click on a link in the sent e-mail.
3. See error message "Link not found.".

### Expected behaviour
Links should not throw 404. 

```
yii\web\NotFoundHttpException: Link not found. in /var/www/vendor/putyourlightson/craft-campaign/src/controllers/TrackerController.php:81
Stack trace:
#0 [internal function]: putyourlightson\campaign\controllers\TrackerController->actionClick()
#1 /var/www/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#2 /var/www/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams()
#3 /var/www/vendor/craftcms/cms/src/web/Controller.php(189): yii\base\Controller->runAction()
#4 /var/www/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction()
#5 /var/www/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction()
#6 /var/www/vendor/craftcms/cms/src/web/Application.php(577): craft\web\Application->runAction()
#7 /var/www/vendor/craftcms/cms/src/web/Application.php(253): craft\web\Application->_processActionRequest()
#8 /var/www/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest()
#9 /var/www/web/index.php(21): yii\base\Application->run()
#10 {main}
```

### Versions
- Plugin version: 1.16.1
- Craft version: 3.5.12.1

**Comment by bencroker** [comment]
It sounds like the link is invalid or may have been deleted. What would you expect to happen in this situation?

**Comment by Saboteur777** [comment]
Okay, I dug into code and realized there should be a `lid` parameter on the link. It is there, but the `=` is missing, that's why the `LinkRecord` is not found: 

`/actions/campaign/t/click?cid=c5f19bcd45271fyHJ&sid=s5f7e116f8380bpai&lidl5f7d88fcb8ef6msm`

**Comment by bencroker** [comment]
So is the URL you received incorrect? Perhaps miscopied or manipulated by the email client?

**Comment by Saboteur777** [comment]
It is incorrect as the $lid parameter is appended without the `=`, so the parameter won't be found by the TrackerController, hence the 404. 

I am using Mailhog for local testing, so no email client is in the equation.

**Comment by Saboteur777** [comment]
![output](https://user-images.githubusercontent.com/9061795/95379560-d3dde600-08e5-11eb-8a59-e3a4010a01cd.gif)

**Comment by bencroker** [comment]
This appears to be caused by a bug introduced in Craft 3.5.12, see https://github.com/craftcms/cms/issues/6900#issuecomment-705166068.

**Comment by Saboteur777** [comment]
Thanks for going after this, this is so exciting! 😀

**Comment by bencroker** [comment]
I decided to fix this in https://github.com/putyourlightson/craft-campaign/commit/b2c6e467ff37a5508ec71e14634cf6aae16cecfc after all as this is a rather nasty bug and I cannot know if and when it will be fixed in Craft. Released in version 1.16.2, thanks for reporting it!

**Comment by Saboteur777** [comment]
Thanks a lot!
